### PR TITLE
[HOU-2020] Added second group of speakers and updated program

### PR DESCRIPTION
**Pull request by coolblknerd**

### CODE REVIEW

File: data/events/2020-houston.yml [hunk 1]
```diff
@@ -183,4 +183,212 @@ sponsor_levels:
...
+    date: 2020-04-15
+    start_time: "09:55"
+    end_time: "10:10"
+    background_color: "#D0D0D0"
+  - title: "michael-nir"
+    type: talk
+    date: 2020-04-15
+    start_time: "10:20"
+    end_time: "10:50"
+  - title: "paul-meharg"
```
Comment (adrianmoisey) [inline data/events/2020-houston.yml]: Should this be `paul-mehard` ?

File: data/events/2020-houston.yml [hunk 2]
```diff
@@ -183,4 +183,212 @@ sponsor_levels:
...
+    date: 2020-04-15
+    start_time: "08:00"
+    end_time: "09:00"
+  - title: "Opening Welcome Day 2"
+    type: custom
+    date: 2020-04-15
+    start_time: "09:00"
+    end_time: "09:15"
+    background_color: "#D0D0D0"
+  - title: "maria-oneill"
```
Comment (adrianmoisey) [inline data/events/2020-houston.yml]: Should this be `maria-f-oneill` ?

File: data/events/2020-houston.yml [hunk 3]
```diff
@@ -183,4 +183,212 @@ sponsor_levels:
...
+    type: talk
+    date: 2020-04-14
+    start_time: "13:20"
+    end_time: "13:50"
+  - title: "sabine-khan"
+    type: talk
+    date: 2020-04-14
+    start_time: "16:00"
+    end_time: "16:30"
+  - title: "matty-stratton"
```
Comment (adrianmoisey) [inline data/events/2020-houston.yml]: `matt-stratton`

File: content/events/2020-houston/program/jody-wolfborn.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
++++
+Talk_date = "2020-04-15"
+Talk_start_time = "15:40"
+Talk_end_time = "16:35"
+Title = "Secure Your Load: Continuous Security and Compliance in a DevSecOps World"
+Type = "workshop"
```
Comment (adrianmoisey) [inline content/events/2020-houston/program/jody-wolfborn.md]: I thiiiiink that `Type = "workshop"` is causing the title to not appear on the program.

Comment (adrianmoisey) [reply content/events/2020-houston/program/jody-wolfborn.md]: There are a few other occurrences of this.

Comment (adrianmoisey) [reply content/events/2020-houston/program/jody-wolfborn.md]: It should be `Type = "talk"`

**Comment by bridgetkromhout** [comment]
Hi, @coolblknerd - yesterday @skryukova submitted changes and your PR has a number of conflicts with them. Please merge upstream/master into your coolblknerd:htx-2020-feb-add-speakers-2 branch so you can resolve these conflicts, and then you can re-open this pull request.

**Pull request closed without merging**